vmm/src/vm_config.rs

```diff
@@ -550,6 +550,8 @@ pub struct PayloadConfig {
     pub cmdline: Option<String>,
     #[serde(default)]
     pub initramfs: Option<PathBuf>,
+    #[serde(default)]
+    pub igvm: Option<PathBuf>,
```

Comment:
Ditto. Need to be behind the `igvm` feature gate.

Reply:
I have tried that but got fuzz error from linux_loader_cmdline.rs as feature is not passed to fuzz

Reply:
@likebreath  Do you recommend to use igvm feature in fuzz directory as well?

Reply:
Yes, it's totally fine to add feature gates to the fuzz directory.